### PR TITLE
Add progressively w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by dakshshah96**
Pull request for issue: #10115
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/thinker3197/progressively
 * Official website (optional, not the repository): https://thinker3197.github.io/progressively
 * NPM package url (optional): https://www.npmjs.com/package/progressively
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 17
   - Count of stars: 358
   - Count of forks: 37
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 33
   - Downloads in the last month: 111

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/progressively/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "npmName": "progressively",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "progressively.!(*gz)"
+      ]
+    }
+  ],
+  "version": "1.1.2"
```
Comment (PeterDaveHello) [inline ajax/libs/progressively/package.json]: `version` should be removed

**Comment by dakshshah96** [comment]
@PeterDaveHello Done 👍

**Comment by PeterDaveHello** [comment]
@dakshshah96 thanks

File: ajax/libs/progressively/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "images"
+  ],
+  "npmName": "progressively",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "progressively.!(*gz)"
+      ]
+    }
```
Comment (pvnr0082t) [inline ajax/libs/progressively/package.json]: We need unminified files from `src` directory as well.
```
    {
      "basePath": "src",
      "files": [
        "progressively*"
      ]
    }
```

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍

File: ajax/libs/progressively/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "progressive",
+    "lazy-load",
+    "images"
+  ],
+  "npmName": "progressively",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "progressively.!(*gz)"
```
Comment (PeterDaveHello) [inline ajax/libs/progressively/package.json]: `"**/*.!(*gz)"` is better here.

**Comment by pvnr0082t** [comment]
@PeterDaveHello Okay. I got it.

**Comment by thinker3197** [comment]
@dakshshah96 thanks for picking this up. Would it be ok if you checked the first point on my behalf so _progressively_ could be added to CDNJS website / readme?

**Comment by PeterDaveHello** [comment]
Thank you all!

**Pull request merged**